### ISSUE TITLE
Downloading

### ISSUE THREAD

**Issue by slimsgym** [simplyblk/Fortnitebuilds#45]
i get the error code DNS_PROBE_FINISHED_NXDOMAIN when trying to download any version. while trying to find my own fix, i found something that appears to tell me the domain isnt online, image provided:
![Screenshot 2024-05-19 054719](https://github.com/simplyblk/Fortnitebuilds/assets/170234868/ac9c385f-9435-4e8b-9671-4ec59c5a6768)

**Comment by DrakiJS** [comment]
yeah blk say  he have a backend problem u can check here  https://discord.gg/ZM36jAqt

**Comment by slimsgym** [comment]
> yeah blk say he have a backend problem u can check here https://discord.gg/ZM36jAqt

thanks

**Comment by yossssef34** [comment]
k
